File: sdks/thunkmetrc-go/wrapper/models/lab_tests_type.go
package models

type LabTestsType struct {
    AlwaysPasses bool `json:"AlwaysPasses,omitempty"`
    DependencyMode string `json:"DependencyMode,omitempty"`
    Id int64 `json:"Id,omitempty"`
    InformationalOnly bool `json:"InformationalOnly,omitempty"`
    LabTestResultExpirationDays *int64 `json:"LabTestResultExpirationDays,omitempty"`
    LabTestResultMaximum int64 `json:"LabTestResultMaximum,omitempty"`
    LabTestResultMinimum int64 `json:"LabTestResultMinimum,omitempty"`
    LabTestResultMode string `json:"LabTestResultMode,omitempty"`
    MaxAllowedFailureCount int64 `json:"MaxAllowedFailureCount,omitempty"`
    Name string `json:"Name,omitempty"`
    RequiresTestResult bool `json:"RequiresTestResult,omitempty"`
    ResearchAndDevelopment bool `json:"ResearchAndDevelopment,omitempty"`
}

File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/LabTestsType.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class LabTestsType
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("AlwaysPasses")]
        public required bool AlwaysPasses { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("DependencyMode")]
        public required string DependencyMode { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Id")]
        public required int Id { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("InformationalOnly")]
        public required bool InformationalOnly { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("LabTestResultExpirationDays")]
        public int? LabTestResultExpirationDays { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("LabTestResultMaximum")]
        public required int LabTestResultMaximum { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("LabTestResultMinimum")]
        public required int LabTestResultMinimum { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("LabTestResultMode")]
        public required string LabTestResultMode { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("MaxAllowedFailureCount")]
        public required int MaxAllowedFailureCount { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Name")]
        public required string Name { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("RequiresTestResult")]
        public required bool RequiresTestResult { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("ResearchAndDevelopment")]
        public required bool ResearchAndDevelopment { get; set; }
    }
}
